### ISSUE TITLE
Documentation of SetVolume is wrong

### ISSUE THREAD

**Issue by malliaridis** [Palakis/obs-websocket#646]
##### Issue type
- Bug report

##### Description
The documentation of `SetVolume` defines a volume value of type mul between `0.0` and `20.0`. Trying that out throws an error. Values are accepted only between `0.0` and `1.0` of type mul.

Similar problem occurs on values of type dB. Accepted values defined by protocol are between `0.0` and `26.0`. Values however need to be here negative or `0.0` whereas values strict smaller `-96.X` are already treated as `-inf` by OBS.

##### Steps to reproduce and other useful info
1. Send `SetVolume` request with volumes above 0.0
2. See response error

##### Logs
```
13:40:16.746: [obs-websocket] Request >> '{"source":"Arctis 7 Chat","useDecibel":true,"volume":-100.0,"message-id":"-350784357","request-type":"SetVolume"}'
13:40:16.746: [obs-websocket] Update << '{
13:40:16.746:     "sourceName": "Arctis 7 Chat",
13:40:16.746:     "update-type": "SourceVolumeChanged",
13:40:16.746:     "volume": 9.9999997473787516e-6
13:40:16.746: }'
13:40:16.746: [obs-websocket] Response << '{
13:40:16.746:     "message-id": "-350784357",
13:40:16.746:     "status": "ok"
13:40:16.746: }'
13:44:53.653: [obs-websocket] Request >> '{"source":"Arctis 7 Chat","useDecibel":false,"volume":1.5,"message-id":"-269773531","request-type":"SetVolume"}'
13:44:53.653: [obs-websocket] Response << '{
13:44:53.653:     "error": "invalid request parameters",
13:44:53.653:     "message-id": "-269773531",
13:44:53.653:     "status": "error"
13:44:53.653: }'
13:47:17.858: [obs-websocket] Request >> '{"source":"Arctis 7 Chat","useDecibel":true,"volume":1.5,"message-id":"-1115414371","request-type":"SetVolume"}'
13:47:17.859: [obs-websocket] Response << '{
13:47:17.859:     "error": "invalid request parameters",
13:47:17.859:     "message-id": "-1115414371",
13:47:17.859:     "status": "error"
13:47:17.859: }'
13:47:32.716: [obs-websocket] Request >> '{"source":"Arctis 7 Game","useDecibel":true,"volume":1.0,"message-id":"-2038134694","request-type":"SetVolume"}'
13:47:32.716: [obs-websocket] Response << '{
13:47:32.716:     "error": "invalid request parameters",
13:47:32.716:     "message-id": "-2038134694",
13:47:32.716:     "status": "error"
13:47:32.716: }'
13:47:41.308: [obs-websocket] Request >> '{"source":"Arctis 7 Game","useDecibel":true,"volume":0.0,"message-id":"185103804","request-type":"SetVolume"}'
13:47:41.308: [obs-websocket] Update << '{
13:47:41.308:     "sourceName": "Arctis 7 Game",
13:47:41.308:     "update-type": "SourceVolumeChanged",
13:47:41.308:     "volume": 1.0
13:47:41.308: }'
13:47:41.308: [obs-websocket] Response << '{
13:47:41.308:     "message-id": "185103804",
13:47:41.308:     "status": "ok"
13:47:41.308: }'
13:47:46.724: [obs-websocket] Request >> '{"source":"Arctis 7 Chat","useDecibel":true,"volume":0.0,"message-id":"-580248008","request-type":"SetVolume"}'
13:47:46.725: [obs-websocket] Update << '{
13:47:46.725:     "sourceName": "Arctis 7 Chat",
13:47:46.725:     "update-type": "SourceVolumeChanged",
13:47:46.725:     "volume": 1.0
13:47:46.725: }'
13:47:46.725: [obs-websocket] Response << '{
13:47:46.725:     "message-id": "-580248008",
13:47:46.725:     "status": "ok"
13:47:46.725: }'
13:48:01.344: [obs-websocket] Request >> '{"source":"Arctis 7 Chat","useDecibel":true,"volume":-99.5,"message-id":"41709973","request-type":"SetVolume"}'
13:48:01.344: [obs-websocket] Update << '{
13:48:01.344:     "sourceName": "Arctis 7 Chat",
13:48:01.344:     "update-type": "SourceVolumeChanged",
13:48:01.344:     "volume": 1.0592539183562621e-5
13:48:01.344: }'
13:48:01.344: [obs-websocket] Response << '{
13:48:01.344:     "message-id": "41709973",
13:48:01.344:     "status": "ok"
13:48:01.344: }'
13:48:24.929: [obs-websocket] Request >> '{"source":"Arctis 7 Chat","useDecibel":true,"volume":-99.0,"message-id":"-2140328232","request-type":"SetVolume"}'
13:48:24.929: [obs-websocket] Update << '{
13:48:24.929:     "sourceName": "Arctis 7 Chat",
13:48:24.929:     "update-type": "SourceVolumeChanged",
13:48:24.929:     "volume": 1.1220189662708435e-5
13:48:24.929: }'
13:48:24.929: [obs-websocket] Response << '{
13:48:24.929:     "message-id": "-2140328232",
13:48:24.929:     "status": "ok"
13:48:24.929: }'
13:48:31.050: [obs-websocket] Update << '{
13:48:31.050:     "sourceName": "Arctis 7 Chat",
13:48:31.050:     "update-type": "SourceVolumeChanged",
13:48:31.050:     "volume": 1.1220184564590454
13:48:31.050: }'
13:48:32.004: [obs-websocket] Update << '{
13:48:32.004:     "sourceName": "Arctis 7 Chat",
13:48:32.004:     "update-type": "SourceVolumeChanged",
13:48:32.004:     "volume": 1.0
13:48:32.004: }'
13:48:32.517: [obs-websocket] Update << '{
13:48:32.517:     "sourceName": "Arctis 7 Chat",
13:48:32.517:     "update-type": "SourceVolumeChanged",
13:48:32.517:     "volume": 1.0
13:48:32.517: }'
13:48:36.449: [obs-websocket] Request >> '{"source":"Arctis 7 Chat","useDecibel":true,"volume":-99.0,"message-id":"216705751","request-type":"SetVolume"}'
13:48:36.449: [obs-websocket] Update << '{
13:48:36.449:     "sourceName": "Arctis 7 Chat",
13:48:36.449:     "update-type": "SourceVolumeChanged",
13:48:36.449:     "volume": 1.1220189662708435e-5
13:48:36.449: }'
13:48:36.449: [obs-websocket] Response << '{
13:48:36.449:     "message-id": "216705751",
13:48:36.449:     "status": "ok"
13:48:36.449: }'
13:48:55.635: [obs-websocket] Request >> '{"source":"Arctis 7 Chat","useDecibel":true,"volume":-90.0,"message-id":"1285947847","request-type":"SetVolume"}'
13:48:55.635: [obs-websocket] Update << '{
13:48:55.635:     "sourceName": "Arctis 7 Chat",
13:48:55.635:     "update-type": "SourceVolumeChanged",
13:48:55.635:     "volume": 3.1622777896700427e-5
13:48:55.635: }'
13:48:55.635: [obs-websocket] Response << '{
13:48:55.635:     "message-id": "1285947847",
13:48:55.635:     "status": "ok"
13:48:55.635: }'
13:49:04.682: [obs-websocket] Request >> '{"source":"Arctis 7 Chat","useDecibel":true,"volume":-95.0,"message-id":"-417418385","request-type":"SetVolume"}'
13:49:04.682: [obs-websocket] Update << '{
13:49:04.682:     "sourceName": "Arctis 7 Chat",
13:49:04.682:     "update-type": "SourceVolumeChanged",
13:49:04.682:     "volume": 1.7782793293008581e-5
13:49:04.682: }'
13:49:04.682: [obs-websocket] Response << '{
13:49:04.682:     "message-id": "-417418385",
13:49:04.682:     "status": "ok"
13:49:04.682: }'
13:49:13.308: [obs-websocket] Request >> '{"source":"Arctis 7 Chat","useDecibel":true,"volume":-97.0,"message-id":"-1842629942","request-type":"SetVolume"}'
13:49:13.308: [obs-websocket] Update << '{
13:49:13.308:     "sourceName": "Arctis 7 Chat",
13:49:13.308:     "update-type": "SourceVolumeChanged",
13:49:13.308:     "volume": 1.412537858413998e-5
13:49:13.308: }'
13:49:13.308: [obs-websocket] Response << '{
13:49:13.308:     "message-id": "-1842629942",
13:49:13.308:     "status": "ok"
13:49:13.308: }'
13:49:22.616: [obs-websocket] Request >> '{"source":"Arctis 7 Chat","useDecibel":true,"volume":-96.0,"message-id":"-568517865","request-type":"SetVolume"}'
13:49:22.616: [obs-websocket] Update << '{
13:49:22.616:     "sourceName": "Arctis 7 Chat",
13:49:22.616:     "update-type": "SourceVolumeChanged",
13:49:22.616:     "volume": 1.5848925613681786e-5
13:49:22.616: }'
13:49:22.616: [obs-websocket] Response << '{
13:49:22.616:     "message-id": "-568517865",
13:49:22.616:     "status": "ok"
13:49:22.616: }'
13:49:35.906: [obs-websocket] Request >> '{"source":"Arctis 7 Chat","useDecibel":true,"volume":-96.5,"message-id":"-255326741","request-type":"SetVolume"}'
13:49:35.906: [obs-websocket] Update << '{
13:49:35.906:     "sourceName": "Arctis 7 Chat",
13:49:35.906:     "update-type": "SourceVolumeChanged",
13:49:35.906:     "volume": 1.496236291131936e-5
13:49:35.906: }'
13:49:35.906: [obs-websocket] Response << '{
13:49:35.906:     "message-id": "-255326741",
13:49:35.906:     "status": "ok"
13:49:35.906: }'
13:49:48.587: [obs-websocket] Request >> '{"source":"Arctis 7 Chat","useDecibel":true,"volume":-96.2,"message-id":"-703462485","request-type":"SetVolume"}'
13:49:48.587: [obs-websocket] Update << '{
13:49:48.587:     "sourceName": "Arctis 7 Chat",
13:49:48.587:     "update-type": "SourceVolumeChanged",
13:49:48.587:     "volume": 1.5488169083255343e-5
13:49:48.587: }'
13:49:48.587: [obs-websocket] Response << '{
13:49:48.587:     "message-id": "-703462485",
13:49:48.587:     "status": "ok"
13:49:48.587: }'
13:50:01.011: [obs-websocket] Request >> '{"source":"Arctis 7 Chat","useDecibel":true,"volume":-96.01,"message-id":"-1553079275","request-type":"SetVolume"}'
13:50:01.011: [obs-websocket] Update << '{
13:50:01.011:     "sourceName": "Arctis 7 Chat",
13:50:01.011:     "update-type": "SourceVolumeChanged",
13:50:01.011:     "volume": 1.5830699339858256e-5
13:50:01.011: }'
13:50:01.011: [obs-websocket] Response << '{
13:50:01.011:     "message-id": "-1553079275",
13:50:01.011:     "status": "ok"
13:50:01.011: }'
13:50:09.942: [obs-websocket] Request >> '{"source":"Arctis 7 Chat","useDecibel":true,"volume":-96.1,"message-id":"-51160335","request-type":"SetVolume"}'
13:50:09.942: [obs-websocket] Update << '{
13:50:09.942:     "sourceName": "Arctis 7 Chat",
13:50:09.942:     "update-type": "SourceVolumeChanged",
13:50:09.942:     "volume": 1.5667517800466157e-5
13:50:09.942: }'
13:50:09.942: [obs-websocket] Response << '{
13:50:09.942:     "message-id": "-51160335",
13:50:09.942:     "status": "ok"
13:50:09.942: }'
```

##### Technical information
- **Operating System** : Windows 10 64bit
- **OBS Studio version** : 26.1.0 (64bit)
- **obs-websocket version** : 4.8

**Comment by malliaridis** [comment]
Is #629 related to it?

**Comment by tt2468** [comment]
This is because you are on v4.8 and there was a recent change which increased the maximums, which will be included in v4.9

**Comment by malliaridis** [comment]
My bad, I am working usually with the 4.X branch protocol which often has changes not included in 4.8. I'll better keep that in mind.  😅

**Comment by tt2468** [comment]
Yeah and if you're ever wondering, just view the git blame and see when it was changed.
